apps/www/src/content/docs/components/combobox.md

```diff
@@ -62,7 +62,7 @@ const value = ref({})
         :aria-expanded="open"
         class="w-[200px] justify-between"
       >
-        {{ value ? frameworks.find((framework) => framework.value === value)?.label : 'Select framework...' }}
+        {{ value ? frameworks.find((framework) => framework.value === value?.value)?.label : 'Select framework...' }}
```

Comment:
based on [`ComboboxDemo.vue`](https://github.com/radix-vue/shadcn-vue/blob/dev/apps/www/src/lib/registry/default/example/ComboboxDemo.vue)

There is no need to use `.value` in `<template>`

```suggestion
        {{ value ? frameworks.find((framework) => framework.value === value)?.label : 'Select framework...' }}
```